scripts/jest/config.react-dom-test-suite.js

```diff
@@ -0,0 +1,23 @@
+const moduleNameMapper = {
+  '^react-dom$': '<rootDir>/lib/index.js',
+  '^react$': '<rootDir>/node_modules/react/',
+  '^react-reconciler$': '<rootDir>/node_modules/react-reconciler',
+  '^react-dom/test-utils$': '<rootDir>/node_modules/react-dom/test-utils',
+};
+
+module.exports = Object.assign(
+  {},
+  {
+    rootDir: process.cwd(),
+    roots: ['<rootDir>'],
+    moduleNameMapper,
+    testRegex:
+      'test/react-test-suite/packages/react-dom/src/__tests__/.*\\.js$',
```

Comment:
theorectically there are tests in `react-shared` as well that are pertinent but i'm not sure it's worth trying to figure out which are.